app/Http/Controllers/MissedApproachController.php

```diff
@@ -28,4 +31,17 @@ public function create(CreateMissedApproachNotification $request): JsonResponse
             return response()->json(['message' => $alreadyActiveException->getMessage()], 409);
         }
     }
+
+    public function acknowledge(
+        AcknowledgeMissedApproach $request,
+        MissedApproachNotification $missedApproachNotification
+    ): JsonResponse {
+        try {
+            $this->service->acknowledge($missedApproachNotification, $request->validated()['remarks']);
+        } catch (CannotAcknowledgeMissedApproachException $cannotAcknowledge) {
+            return response()->json(['message' => 'You cannot acknowledge this missed approach'])->setStatusCode(403);
+        }
```

Comment:
Is there a way perhaps to know this ahead of time and move the authorisation logic to the form request?

Reply:
Or maybe some of the base checks are dealt with e.g. are they controlling in the form request?